arch/x86/kvm/pkvm/pgtable.c

```diff
@@ -694,3 +717,38 @@ void pkvm_pgtable_lookup(struct pkvm_pgtable *pgt, unsigned long vaddr,
 	if (level)
 		*level = data.level;
 }
+
+/**
+ * pkvm_pgtable_set_owner() - Set the owner ID annotation for a range of virtual
```

Comment:
It might be better to define pkvm_pgtable_set_owner() after pkvm_pgtable_unmap(), not after pkvm_pgtable_lookup()? (set_owner is similar to map and unmap since it modifies PTEs, while lookup doesn't modify anything.)